### ISSUE TITLE
Tool fails to flash device with encryption enabled

### ISSUE THREAD

**Issue by nicoloboatto** [mongoose-os/mos#16]
I haven't used the tool in a couple of months, after having updated to the latest version I now run into a problem while flashing devices with encryption.

I tried both the release and the latest channels.

This is the output from mos:

```bash
Using port /dev/ttyUSB0
Opening /dev/ttyUSB0 @ 115200...
Connecting to ESP32 ROM, attempt 1 of 10...
Connecting to ESP32 ROM, attempt 2 of 10...
Connecting to ESP32 ROM, attempt 3 of 10...
Connecting to ESP32 ROM, attempt 4 of 10...
  Connected, chip: ESP32D0WDQ6 R1
Device MAC address: --REDACTED--
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xb85fc6]

goroutine 1 [running]:
github.com/mongoose-os/mos/mos/flash/esp32.(*Fuse).IsWritable(0x0, 0xc000279a70)
	/build/mos-LHsKMY/mos-2.14.0+71e2991~bionic0/go/src/github.com/mongoose-os/mos/mos/flash/esp32/efuses.go:237 +0x26
github.com/mongoose-os/mos/mos/flash/esp32.(*Fuse).SetValue(0x0, 0xc000279d28, 0xc000279d28, 0x12)
	/build/mos-LHsKMY/mos-2.14.0+71e2991~bionic0/go/src/github.com/mongoose-os/mos/mos/flash/esp32/efuses.go:302 +0x43
main.esp32GenKey(0x1290a20, 0xc0000c4020, 0x0, 0x0, 0x0, 0x0)
	/build/mos-LHsKMY/mos-2.14.0+71e2991~bionic0/go/src/github.com/mongoose-os/mos/mos/esp32_gen_key.go:122 +0xdde
main.run(0xc0000c9860, 0x1290a20, 0xc0000c4020, 0x0, 0x0, 0x1, 0x1)
	/build/mos-LHsKMY/mos-2.14.0+71e2991~bionic0/go/src/github.com/mongoose-os/mos/mos/main.go:195 +0xa2
main.main()
	/build/mos-LHsKMY/mos-2.14.0+71e2991~bionic0/go/src/github.com/mongoose-os/mos/mos/main.go:297 +0x5a9
```

**Comment by rojer** [comment]
this was fixed by https://github.com/mongoose-os/mos/commit/ff7a5cb20e83730bb800cb42e612ef042d2f6d8c

i just pushed mos-latest which includes this fix (version 201906271728)

**Comment by nicoloboatto** [comment]
I just saw the last commit and built from source, thank you!
